### DISCUSSION TITLE
Différence d'index entre local et ftp

### DISCUSSION THREAD

**Discussion by artesburger** [dadoonet/fscrawler#1504] [General]
Bonjour,

J'ai remarqué en indexant un répertoire local (6giga). Et en indexant le même en ftp qui se trouve sur un nas, que je ne possède pas le même nombre de docs.
En ftp, fscrawler contient plus d'erreur, et s'arrête d'un seul coup et se déconnecte.
A la fin j'ai 32 000 fichiers sur le répertoire local d'indexé, et 3000 sur celui en ftp.
Avez-vous une idée d'ou provient ce problème ?

**Comment by dadoonet** [comment]
> En ftp, fscrawler contient plus d'erreur, et s'arrête d'un seul coup et se déconnecte.

Il faudrait que tu fournisses les logs en question.
Peut-être faut-il implémenter un système de "keep-alive" sur la connection FTP ?

Et sans doute un mécanisme de checkpoint avec des retry... Voir #1098

**Reply by dadoonet** [reply]
> Mais du coup comment dire à fscawler de "keep-alive" la connection ftp ?

On ne peut pas. Cette option n'est pas implémentée.

**Comment by dadoonet** [comment]
Merci de poster les logs au format texte et non image. C'est difficile à lire, non indexé par les moteurs de recherche et éventuellement incomplet.

merciz
